recipes/rb-gli/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "gli" %}
+{% set version = "2.19.0" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+  sha256: 31dd578a13461cf53e47c8811e7e69b10fed8d4e50d3a38fd230a137223a70cb
+
+build:
+  noarch: generic
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: {{ win }}
+
+requirements:
+  host:
+    - ruby
+  run:
+    - ruby
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: Apache-2.0
+  license_family: Apache
+  license_file: {{ name }}-{{ version }}/LICENSE.txt
+  summary: |
+    Build command-suite CLI apps that are awesome.
+    Bootstrap your app, add commands, options and documentation while
+    maintaining a well-tested idiomatic command-line app
+
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/davetron5000/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - conda-forge/ruby
```

Comment:
I am not sure this is working yet, if it is we need to confirm with @scopatz and @ jimmynguyc if they want to be included in all ruby recipes.